web/less/variables.less

```diff
@@ -71,6 +71,87 @@
 @grey5: #686868;
 @grey6: #31313d;
 
+@VeryLightGrey: #CFCFCF;
+@lightGrey: #404040;
+@lightGrey1: #d7d7d7;
+@lightGrey2: #E8E8E8;
+@lightGrey3: #e7e7e7;
+
+@BaliHaiGrey: #8E99A5;
```

Comment:
For consistency, shouldn't these all use lower camel case, e.g. `@baliHaiGrey`?

Reply:
updated in new commit 
`b7ea82d -> 8730cfa`

Reply:
Never mind. I was looking at generated files.